2200069/0120_06.py

```diff
@@ -0,0 +1,14 @@
+"""최빈수구하기"""
+
+for i in range(int(input())):
+    caseNum = int(input())
+    scoreLi = list(map(int, input().split()))
+    countLi = []
+    for j in scoreLi:
+        countLi.append(scoreLi.count(j))
+        if max(countLi) == scoreLi.count(j):
+            prnt = j
+        else:
+            pass
```

Comment:
```suggestion
        if max(countLi) == scoreLi.count(j):
            prnt = j
```
`pass`는 아무 역할도 수행하지 않으므로, `else:pass`는 지워주셔도 될 것 같습니다!